### DISCUSSION TITLE
The model seem working normally inside the web page editor, but it not working to me while try to do with vscode studio

### DISCUSSION THREAD

**Discussion by 13448-Chen** [google/model-viewer#3882] [Q&A]
I have tried to put this little model on my web(haven't launched yet) with using vscode studio,  it seem it couldnt work and i dont know why is that. The model just ran normally on the model viewer webpage editor.

I created the model using blender, I decreased the number of triangles to below 10,000 units, and it still doesn't work. 

can any one help me with this🙏

<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
  <title>Document</title>

  <style>
    .box {display: flex;}
    model-viewer {width: 800px; height: 600px; margin: 0, auto;}
  </style>
</head>

<body>

  <div class="box">
    <model-viewer src="badge5.gltf" ar ar-modes="webxr scene-viewer quick-look" camera-controls poster="poster.webp" shadow-intensity="1">
        <div class="progress-bar hide" slot="progress-bar">
            <div class="update-bar"></div>
        </div>
    </model-viewer>
    
</body>
</html>

![Screenshot 2022-10-15 at 12 09 34 AM](https://user-images.githubusercontent.com/80742889/195892505-2e0c95b0-c6b7-4bf8-b40d-3d31122a28f6.png)
![Screenshot 2022-10-15 at 12 09 50 AM](https://user-images.githubusercontent.com/80742889/195892525-3e81525b-0be3-47d5-8001-58ed9af5d25d.png)
![Screenshot 2022-10-15 at 12 10 12 AM](https://user-images.githubusercontent.com/80742889/195892527-f6c51d96-31a5-42ba-88a8-86367dc483bb.png)

**Comment by elalish** [comment]
Look in the JS console for errors/warnings. I would guess an incorrect path, or some CSS problem.
